Description: Fix test to also ignore options renamed in lt 2.5.3
Author: Andreas Metzler <ametzler@debian.org>
Bug-Debian: https://bugs.debian.org/1092416
Origin: vendor
Forwarded: no
Last-Update: 2025-01-11

--- wmaker-0.96.0.orig/Makefile.am
+++ wmaker-0.96.0/Makefile.am
@@ -143,7 +143,9 @@ configure-documentation:
 		--ignore-prg 'disable-option-checking,enable-fast-install # should be in INSTALL' \
 		--ignore-prg 'disable-libtool-lock,with-pic,with-gnu-ld,with-sysroot # for libtool' \
 		--ignore-prg 'runstatedir #new in autoconf 2.70, backported in Debian' \
-		--ignore-prg 'with-aix-soname #new in libtool 2.4.4' \
+		--ignore-prg 'with-aix-soname #new in libtool 2.4.4-2.5.2' \
+		--ignore-prg 'enable-aix-soname #new in libtool 2.5.3' \
+		--ignore-prg 'enable-pic #new in libtool 2.5.3' \
 		--ignore-prg 'with-x # no use, it would not work without X'
 
 .PHONY: configure-documentation
